src/screens/ProfileScreen/index.js

```diff
@@ -172,7 +174,6 @@ const ProfileScreen = ({route}) => {
   const [isLoadingUpdateImageCamera, setIsLoadingUpdateImageCamera] = React.useState(false);
   const [errorChangeRealName, setErrorChangeRealName] = React.useState('');
   const [postOffset, setPostOffset] = React.useState(0);
-  const [isLastPage, setIsLastPage] = React.useState(false);
   const [loading, setLoading] = React.useState(false);
   const [isPostOptionModalOpen, setIsOptionModalOpen] = React.useState(false);
   const [selectedPostForOption, setSelectedPostForOption] = React.useState(null);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The state variable `isLastPage` has been removed in the new hunk. This variable was previously used to determine if the user had reached the end of the feed. However, it seems that this functionality has been replaced by the `fetchNextFeeds` function from the `useFeedPreloadHook`. If this is not the case and there's still a need for `isLastPage`, you should consider adding it back.



<!-- This is an auto-generated comment by OSS CodeRabbit -->

Reply:
is last page is not needed anymore